nixos/tests/installer.nix

```diff
@@ -648,8 +652,13 @@ let
           in [
             (pkgs.grub2.override { inherit zfsSupport; })
             (pkgs.grub2_efi.override { inherit zfsSupport; })
+            pkgs.nixos-artwork.wallpapers.simple-dark-gray-bootloader
+            pkgs.perlPackages.FileCopyRecursive
+            pkgs.perlPackages.XMLSAX
+            pkgs.perlPackages.XMLSAXBase
```

Comment:
This is a little confusing to me; this test should have already been using `switch-to-configuration-ng`, so why did it not need the Perl dependencies already?

Reply:
They were pulled in by installer stc -> installer bootloader install script -> is grub by default lololololololofucking kill me